zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/FastCodeUpdates.kt

```diff
@@ -0,0 +1,56 @@
+package app.cash.zipline.loader
```

Comment:
supernit: copyright

(I think we might be due to get Spotless managing this?)

Reply:
I'll do this. I just had to do it for paraphrase and will be updating redwood and friends with the new setup.